integration/dev_test.go

```diff
@@ -322,3 +323,37 @@ func createModifiedKubeconfig(namespace string) ([]byte, string, error) {
 	yaml, err := clientcmd.Write(*kubeConfig)
 	return yaml, contextName, err
 }
+
+func TestDevNoDeployer(t *testing.T) {
+	if testing.Short() || RunOnGCP() {
+		t.Skip("skipping kind integration test")
```

Comment:
I'm confused by this message. How do you know that this is authoritatively kind?